fp.js

```diff
@@ -32,3 +47,8 @@ var people = [
   {name: "Jasmine", age: 39}
 ]
 var peopleWhoseNamesHaveTheLetterS = people.filter(function(person){return person.name.match("s")});
+
+// Answer:
+var sNames = people.forEach(function(person){
```

Comment:
`.forEach` doesn't have a return value so we'd need to push onto our array inside the loop:
```js
let peopleWhoseNamesHaveTheLetterS = []
people.forEach(function(person){
  if(person.name.match("s"))
    peopleWhoseNamesHaveTheLetterS.push(person)
  })
```